### PR TITLE
Fix undeclared init0(bool) with some IMU boards

### PR DESCRIPTION
**Pull request by btry**
without that : compilation fails. Fix based on line 589 in FreeIMU.cpp

### CODE REVIEW

**Comment by mjs513** [comment]
Can agree to everything except the Curie_IMU.  This IMU compiles fine.  Not sure how to do the edit on your merge request.

File: libraries/FreeIMU/FreeIMU.h [hunk 1]
```diff
@@ -509,7 +509,7 @@ class FreeIMU
     #if HAS_ITG3200()
 		void init(bool fastmode);
 		void init(int acc_addr, int gyro_addr, bool fastmode);
-	#elif HAS_ALTIMU10() || HAS_LSM9DS0()
+	#elif HAS_ALTIMU10() || HAS_LSM9DS0() || HAS_ADA_10_DOF() || HAS_CURIE() || HAS_LSM9DS1() || HAS_TPS()
```
Comment (mjs513) [inline libraries/FreeIMU/FreeIMU.h]: #elif HAS_ALTIMU10() || HAS_LSM9DS0() || HAS_ADA_10_DOF() ||  HAS_LSM9DS1() || HAS_TPS()

**Comment by btry** [comment]
Ok, I removed HAS_CURIE()

**Comment by btry** [comment]
Actually, your comment about compilation for Curie extends to TPS, because they have their own init sequence. However it is the same as ALTMIU10, Adafruit 10DOF and boards with similar chips. Then I submit merge of these distinct blocks.

**Comment by mjs513** [comment]
Merge Completed.  Thanks for the updates,

Mike

**Pull request merged**